extension/globals.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+// the root of the CDN. This is also (unfortunately) hardcoded in
+// event-page.html
+//const cdn = 'http://localhost:5001/dev';
```

Comment:
If we leave this, perhaps add a comment above it noting that for local development eng will frequently want to swap this in for now. How should we handle this in the longer term, what do other extensions do?

Reply:
Actually it looks like my last reference to cdn went away? I'm not sure how, it may need to come back, but for now I'll delete it. Good question!